docs/INTERACTIVE_DEMO_RUNBOOK.md

```
@@ -0,0 +1,728 @@
+
+---
+
+### Step 1.3: Verify NATS WebSocket
+
+**Command:**
+```bash
+curl -s --http1.1 -H "Connection: Upgrade" -H "Upgrade: websocket" -H "Sec-WebSocket-Key: test" -H "Sec-WebSocket-Version: 13" http://localhost:9223/ -o /dev/null -w "%{http_code}"
+```
+
+**Expected Output:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)